website/resources/components/ml.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus ML
```

Comment:
How is this different from the `azure-ml.md` file? Isn't this the same component?

Reply:
No, the Arcus ML component has more general ML functionality like image shading, conversion, time operations... while the Arcus Azure ML component really focuses on the connection with Azure like triggering trainings on Azure ML.